packages/react-composites/src/composites/CallComposite/adapter/AzureCommunicationCallAdapter.ts

```diff
@@ -182,7 +183,10 @@ class CallContext {
         page: newPage,
         endedCall: latestEndedCall,
         devices: clientState.deviceManager,
-        latestErrors: clientState.latestErrors
+        latestErrors: clientState.latestErrors,
+        isCameraOn: this.state.call
+          ? !!clientState.calls[this.state.call.id]?.localVideoStreams.find((s) => s.mediaStreamType === 'Video')
```

Comment:
avoid using `this.state` here for the same reason as `getState()`, this function is updating the state so its reasonable to say that the old state is not reliable
(we probably shouldn't be using this.state.page above but that can be a different problem).
We've already calculated the call at the top of the function (L155), can we not use that?